packages/opds-web-client/src/components/Book.tsx

```diff
@@ -27,27 +31,33 @@ export default class Book<P extends BookProps> extends React.Component<P, void>
     // Remove HTML tags from the summary to fit more information into a truncated view.
     // The summary may still contain HTML character entities and needs to be rendered as HTML.
     let summary = (this.props.book && this.props.book.summary && this.props.book.summary.replace(/<\/?[^>]+(>|$)/g, " ")) || "";
+    const bookMedium = this.getMedium(this.props.book);
+    const showMediaIconClass = bookMedium ? "show-media" : "";
+
     return (
-      <div className="book" lang={this.props.book.language}>
+      <div className={`book ${showMediaIconClass}`} lang={this.props.book.language}>
         <CatalogLink
           collectionUrl={this.props.collectionUrl}
           bookUrl={this.props.book.url || this.props.book.id}
           title={this.props.book.title}
           >
           <BookCover book={this.props.book} />
-          <div className="compact-info">
-            <div className="title">{this.props.book.title}</div>
-            { this.props.book.series && this.props.book.series.name &&
-              <div className="series">{this.props.book.series.name}</div>
-            }
-            <div className="authors">
-              {
-                this.props.book.authors.length ?
-                this.props.book.authors.join(", ") :
-                  this.props.book.contributors && this.props.book.contributors.length ?
-                  this.props.book.contributors.join(", ") :
-                  ""
+          <div className={`compact-info ${showMediaIconClass}`}>
+            {this.getMediumSVG(bookMedium, false)}
```

Comment:
I showed this to Mauricio and he suggested putting the title next to this svg in compact view, and hiding it when you hover.

something like:
```
<div class="item-icon"><svg>{the icon here}</svg><span class="brief">Tibetan Peach Pie: A True Account of an Imaginative Life    </span></div>

.item-icon {
    display: flex;
}
.brief {
    white-space: nowrap;
    overflow: hidden;
    text-overflow: ellipsis;
    width: calc(100% - 30px);
    line-height: 1.8;
}
```


![image](https://user-images.githubusercontent.com/3409215/41993947-d74ed56c-7a1a-11e8-9a14-352c9996b0d5.png)

Reply:
I like this. It's not part of @ricardoom 's mockup or the criteria for this ticket so I don't think it should be included, at least not for this iteration, or else we won't be moving along with anything.

Reply:
I'd say I'm not crazy about putting the title in there, I just dont see how its helpful in the collapsed state. Inevitably the title will be truncated unless the card size is doubled, but I dont think that is realistic. I'd say revert back to icon only, and let the user do some hovering and discovering...

Reply:
I don't know that the title is particularly helpful, but I don't like how empty it looks with only the icon.

Also @EdwinGuzman, sorry but I didn't have an opportunity to weigh in on the mockup or the criteria for this ticket until this PR, so I think it's reasonable to have this discussion now.